server/src/main/java/org/apache/iotdb/db/engine/compaction/cross/utils/ReadPointCrossCompactionEstimator.java

```diff
@@ -93,8 +106,17 @@ private long calculateReadingSeqFiles(List<TsFileResource> seqResources) throws
       // it is max aligned series num of one device when tsfile contains aligned series,
       // else is sub compaction task num.
       int concurrentSeriesNum = fileInfo[2] == -1 ? subCompactionTaskNum : fileInfo[2];
-      long seqFileCost =
-          concurrentSeriesNum * (seqResource.getTsFileSize() * fileInfo[1] / fileInfo[0]);
+      long seqFileCost = 0;
+      if (fileInfo[0] == 0) { // If totalChunkNum ==0, i.e. this unSeq tsFile has no chunk.
```

Comment:
Modify the comments as `seq tsFile`

Reply:
Thank your comments.
I have just corrected it.